jekyll/_cci2/configuration-reference.md

```diff
@@ -717,6 +717,14 @@ Key | Required | Type | Description
 requires | N | List | A list of jobs that must succeed for the job to start
 {: class="table table-striped"}
 
+##### **`contexts`**
+Jobs may be configured to use global environment variables set for an organization, see the [Contexts]({{ site.baseurl }}/2.0/workflows) document for adding a context in the application settings.
+
+Key | Required | Type | Description
+----|-----------|------|------------
+context | N | String | The name of the context. The default name is `org-global`.
```

Comment:
The name of the key changes here - under "Jobs" it's referred to as `contexts` and here it's referred to as `context`.